examples/access-token-server/README.md

```diff
@@ -0,0 +1,67 @@
+# Access Token Server
+
+The access token gets google oauth access tokens using a service account. The returned access token can then be used to access google apis.
+
+## Setup for running locally
+
+1. [Create a Google oauth2 service account](https://developers.google.com/identity/protocols/oauth2/service-account)
+2. Grant access to the required api (Google console API & Services Library page for the project created or updated in step 1)
+3. Create a .env file and add contents as shown below.
+4. Encode the json generated when the service account was created by running `yarn encode-credentials`. The script expects the json created in step 1 to be in a file called service_account.json in this directory. It prints a base64 encoded string to the console. Copy this string to the .env file.
+5. Start up the server
+
+## Required environment variables
+
+The server will read an update environment variables from a .env file in this directory. The format the of .env file is as follows
+
+```
+SERVER_PORT=
+LOOKER_SERVER_URL=
+LOOKER_SERVER_VERIFY_SSL=
+LOOKER_CLIENT_ID=
+LOOKER_CLIENT_SECRET=
+SERVICE_ACCOUNT_CREDENTIALS=
```

Comment:
just a heads up, I'll check with Vincent but it might be easier to stick with what we already have defined in their system (just name differences):
```
LOOKERSDK_CLIENT_ID
LOOKERSDK_CLIENT_SECRET
LOOKERSDK_BASE_URL
LOOKERSDK_VERIFY_SSL
GOOGLE_APPLICATION_CREDENTIAL_ENCODED
```

Reply:
to be clear: I'd rather use the new names and I'll ask Vincent how much effort that will be. If he moves us onto AWS Secrets Manager then I'm sure we can change them. If we stay in the ansible vault then probably easiest to use the existing var names